SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityRunNowRequest.java

```diff
@@ -40,6 +42,7 @@ public SingularityRunNowRequest(@JsonProperty("message") Optional<String> messag
                                   @JsonProperty("runId") Optional<String> runId,
                                   @JsonProperty("commandLineArgs") Optional<List<String>> commandLineArgs,
                                   @JsonProperty("resources") Optional<Resources> resources,
+                                  @JsonProperty("s3UploaderAdditionalFiles") List<SingularityS3UploaderFile> s3UploaderAdditionalFiles,
```

Comment:
remember this can be used from the client, so adding args will break the build for folks

Reply:
👍  good catch. Restored the constructor from the most recent release.